tests/e2e/e2e_staking_test.go

```diff
@@ -1,17 +1,32 @@
 package e2e
 
 import (
+	"fmt"
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func (s *IntegrationTestSuite) testStaking(chainEndpoint string, delegatorAddress string, validatorAddressA string, validatorAddressB string, fees sdk.Coin, homePath string) {
+func (s *IntegrationTestSuite) testStaking() {
```

Comment:
```suggestion
func (s *IntegrationTestSuite) TestStaking() {
```

Reply:
bby keeping it lower case it allows it to be called along with all the other tests in `e2e_test.go`.